crates/noirc_evaluator/src/ssa_refactor/acir_gen/acir_ir/generated_acir.rs

```diff
@@ -673,8 +673,20 @@ impl GeneratedAcir {
     /// - `1` if lhs >= rhs
     /// - `0` otherwise
     ///
-    /// See [R1CS Workshop - Section 10](https://github.com/mir-protocol/r1cs-workshop/blob/master/workshop.pdf)
-    /// for an explanation.
+    /// We essentially computes the sign bit of b-a
+    /// For this we sign-extend b-a with c = 2^{max_bits} - (b - a), since both a and b are < 2^{max_bits}
```

Comment:
```suggestion
    /// We essentially computes the sign bit of `b-a`
    /// For this we sign-extend `b-a` with `c = 2^{max_bits} - (b - a)`, since both `a` and `b` are less than `2^{max_bits}`
```

Example of wrapping.

Reply:
Done